exchanges/okcoin/okcoin_websocket.go

```diff
@@ -621,34 +710,85 @@ func (o *OKCoin) GenerateDefaultSubscriptions() ([]stream.ChannelSubscription, e
 	var subscriptions []stream.ChannelSubscription
 	assets := o.GetAssetTypes(true)
 	for x := range assets {
+		if assets[x] != asset.Spot {
+			continue
+		}
```

Comment:
Seems like we can drop the for loop here as no other asset is being used currently? Or is there a pending TODO?

Reply:
No, I left it like that Incase other asset types are added. But, I think it is not necessary.
Removed!